test/CMakeLists.txt

```diff
@@ -186,9 +188,18 @@ ecbuild_add_test( TARGET test_iodaconv_cryosat2
                        "${CMAKE_BINARY_DIR}/bin/cryosat_ice2ioda.py
                        -i testinput/cryosat2_L2_test.nc
                        -o testrun/cryosat2_L2.nc
-                       -d 2019092100"
+                       -d 2019092112"
                       cryosat2_L2.nc)
 
+ecbuild_add_test( TARGET test_iodaconv_cryosat2_DBL
```

Comment:
This test is taking 6-8 minutes to run on my iMac. Not sure why it is so slow - I see that it runs in ~50 seconds during the automated testing.

The 6-8 minute runtime is prohibitive. Would it be acceptable with you make it so that this test doesn't run by default (and perhaps set a variable to make the test run). If this sounds okay with you, you might want to create another script (alongside test/iodaconv_comp.sh) that checks an environment variable and if not set returns true (pass), and if set then runs iodaconv_comp.sh as is done here. That way you can control the test without having to re-run cmake. Once we get the nightly test scheme going then we can place this test into the nightly tests.

Thanks!